gapic/templates/%namespace/%name_%version/%sub/services/%service/client.py.j2

```diff
@@ -81,6 +81,13 @@ class {{ service.client_name }}(metaclass={{ service.client_name }}Meta):
     from_service_account_json = from_service_account_file
 
 
+    {% for message in service.resource_messages -%}
+    @staticmethod
+    def {{ message.name|snake_case }}_path({% for arg in message.resource_path_args %}{{ arg }}: str,{% endfor %}) -> str:
```

Comment:
I think using `(google.api.resource).type` to generate the helper method name is better - It is designed to be the identifier of the resources. And that works for file-level `resource_definition` annotation too.

Reply:
Done